packages/labs/ssr/src/lib/render-value.ts

```diff
@@ -761,7 +859,7 @@ function* renderTemplateResult(
     }
   }
 
-  if (partIndex !== result.values.length) {
+  if (partIndex !== result.values.length && hydratable) {
     throwErrorForPartIndexMismatch(partIndex, result);
```

Comment:
Why are part index mismatches ok with server templates?

Reply:
Ah, I think that was from an old and worse prototype! Thank you for catching this!